packages/skeleton-react/src/components/Progress/Progress.tsx

```diff
@@ -0,0 +1,68 @@
+import { useEffect } from "react";
+import { ProgressProps } from "./types";
+
+export default function Progress({
+	value,
+	min = 0,
+	max = 100,
+	ariaLabelledby = "",
+	// Root
+	base = "overflow-x-hidden",
+	bg = "bg-surface-200-800",
+	height = "h-2",
+	rounded = "rounded",
+	classes = "",
+	// Meter
+	meterBase = "h-full",
+	meterBg = "bg-primary-500",
+	meterRounded = "rounded",
+	meterTransition = "transition-[width]",
+	meterIndeterminateAnim = "indeterminate",
+	meterClasses = "",
+}: ProgressProps) {
+	useEffect(() => {
+		if (min > max) {
+			console.error(
```

Comment:
FYI this does not appear to be triggering for me on either the Svelte or React components.

For Svelte I tested:

1. Setting `value: 150` and `max: 100` in the preview page `$state()` rune
2. Setting the same settings via the inputs on the page
3. Manually passing the values via props

Reply:
Weird, using the playground and just entering any value where this condition would be true results in:
![image](https://github.com/skeletonlabs/skeleton/assets/63101006/895d8c87-ea6b-4426-9b7a-90061e392cf7)

Reply:
Could you confirm this is working as intended?